shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/RewardUpdate.hs

```diff
@@ -290,24 +303,24 @@ rewardStakePool
         pp_nOpt
       }
     )
-  (m1, m2)
+  (RewardAns m1 m2)
   pparams = do
     let hk = _poolId pparams
         blocksProduced = Map.lookup hk b
         actgr@(Stake s) = poolStake hk delegs stake
         Coin pstake = fold s
         sigma = if totalStake == 0 then 0 else fromIntegral pstake % fromIntegral totalStake
         sigmaA = if activeStake == 0 then 0 else fromIntegral pstake % fromIntegral activeStake
-        ls =
+        !ls =
```

Comment:
I think this bang is redundant with the change on line 48 to import `Data.Map.Strict`; the calls to `Map.insert` will force the passed `ls`. (... I think. It's been a while since I 100% understand the strict `Map` interface.)

See module header documentation here https://hackage.haskell.org/package/containers-0.6.4.1/docs/Data-Map-Strict.html